includes/class-wc-calypso-bridge-footer-credits.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC Calypso Bridge Footer Credits
+ */
+class WC_Calypso_Bridge_Footer_Credits {
+
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var object
+	 */
+    protected static $instance = null;
+
+    /**
+	 * Get class instance.
+	 *
+	 * @return object Instance.
+	 */
+	final public static function get_instance(): WC_Calypso_Bridge_Footer_Credits {
+		if ( null === static::$instance ) {
+			static::$instance = new static();
+		}
+
+		return static::$instance;
+	}
+
+    /**
+	 * Constructor.
+	 */
+	public function __construct() {
+		// Bail out early if the current site is not on Woo plan.
+		if ( ! wc_calypso_bridge_is_woo_express_plan() ) {
+			return;
+		}
+
+        return add_filter( 'wpcom_better_footer_credit_link', array( $this, 'get_footer_credits' ) );
+    }
+
+    /**
+     * Get footer credit as HTML.
+     *
+     * @see <wpcomsh/block-theme-footer-credits/class-wpcom-block-theme-footer-credits.php>
+     *
+     * @return string
+     */
+    public function get_footer_credits(): string {
+        $utm_string = '?utm_source=referral&utm_medium=footer-credit&utm_campaign=woo-express-footer-credit';
+        return sprintf( __( 'Powered by <a href="https://woocommerce.com/express/%1$s" rel="nofollow">Woo</a>' ), $utm_string );
```

Comment:
This formatting makes things pretty hard to translate. Typically we'd try to do something like `__( 'Powered by %1$s' )` and then `sprintf` in all of the pieces (vs just the utm_string). That way the translator can't change those details (on purpose or by mistake) and just focuses on the "Powered by <something>" concept. We also wouldn't actually want them to change "Woo" here.

Something like (untested)
```
/* translators: %1$s is replaced with a link to Woo.com */
sprintf( __( 'Powered by %1$s' ), '<a href="https://woocommerce.com/express/' . $utm_string . '" rel="nofollow">Woo</a>' );
```

Also good to include the comment for translators in the line immediately before the string.

Reply:
Good catch, @beaulebens! Updated in https://github.com/Automattic/wc-calypso-bridge/pull/1265/commits/23c195419e7b3a4fe93f8b827b245532ff300a0a